index.ts

```diff
@@ -1398,6 +1453,13 @@ const crypto: { node?: any; web?: any } = {
   web: typeof self === 'object' && 'crypto' in self ? self.crypto : undefined,
 };
 
+const TAGS = ['BIP0340/challenge', 'BIP0340/aux', 'BIP0340/nonce'] as const;
+export type TaggedHashPrefix = typeof TAGS[number];
+/** An object mapping tags to their tagged hash prefix of [SHA256(tag) | SHA256(tag)] */
+const TAGGED_HASH_PREFIXES: { [tag: string]: Uint8Array | undefined } = Object.fromEntries(
```

Comment:
I'm not sure we can support `Object.fromEntries`. Maybe use simpler es2016 alternative etc.